### ISSUE TITLE
Fails to build on Linux with Clang

### ISSUE THREAD

**Issue by GabrielMajeri** [grovesNL/spirv_cross#33]
```
In file included from src/wrapper.cpp:1:
In file included from src/vendor/SPIRV-Cross/spirv_hlsl.hpp:20:
In file included from src/vendor/SPIRV-Cross/spirv_glsl.hpp:20:
In file included from src/vendor/SPIRV-Cross/spirv_cross.hpp:21:
src/vendor/SPIRV-Cross/spirv_common.hpp:22:10: fatal error: 'cstdio' file not found
#include <cstdio>
               ^~~~~~~~
1 error generated.
```

The problem seems to be that the `build.rs` file [enables `libc++`](https://github.com/grovesNL/spirv_cross/blob/8194bbd566b8a9f108850c07e15fd627834eee4f/spirv_cross/build.rs#L9) automatically with Clang, even if it isn't installed.

I am using the latest Clang, but still want to build with the normal GNU `libstdc++`. Is there any specific reason for using `cpp_set_stdlib("c++")`? Right now using G++ works around this problem.

**Comment by grovesNL** [comment]
Thanks for creating the issue! I plan to add more builds to Travis to catch these cases. `cpp_set_stdlib("c++")` is necessary to support Clang on OS X because of SPIRV-Cross dependencies (see [the build failure which prompted this change](https://travis-ci.org/gfx-rs/gfx/jobs/288864985#L808)). We could extend this check to detect the platform as well. I'm hoping we can [feature detect whether this flag is supported upstream in cc-rs](https://github.com/alexcrichton/cc-rs/issues/254) which should make this check more robust in general.

Could you check if Clang on Linux works without the argument?

**Comment by GabrielMajeri** [comment]
Removing the flag from `build.rs` works with no problems using Clang on Linux.

**Comment by grovesNL** [comment]
Fixed in #34
